charts/nexus3/Chart.yaml

```diff
@@ -23,3 +23,4 @@ annotations:
   artifacthub.io/changes: |
     - Added dynamic probe configuration
     - Fixed typo in LDAP configuration
```

Comment:
@albustax the previous changes lines need removing, I know it's not particularly intuitive. Once this is done I'll merge. Thank you very much.

Reply:
np, done !